.github/workflows/auto_cherry_pick.yml

```diff
@@ -43,3 +43,32 @@ jobs:
           labels: |
             Auto_Cherry_Picked
             ${{ matrix.to_branch }}
+
+  create-issue:
+    runs-on: ubuntu-latest
+    if: ${{ always() && contains(join(needs.*.result, ','), 'failure') }}
+    needs: auto-cherry-pick
+    steps:
+      - name: Create Issue on Failed workflow
+        uses: dacbd/create-issue-action@main
+        with:
+          token: ${{ github.token }}
```

Comment:
Use SatQE token instead of GithubToken, refer https://github.com/SatelliteQE/robottelo/blob/master/.github/workflows/auto_cherry_pick.yml#L41